src/main/java/com/gooddata/connector/Integration.java

```diff
@@ -30,7 +30,11 @@
     private final IntegrationProcessStatus runningProcess;
 
     public Integration(final String projectTemplate) {
-        this(projectTemplate, true, null, null, null);
+        this(projectTemplate, true);
+    }
+
+    public Integration(final String projectTemplate, final boolean active) {
+        this(projectTemplate, active, null, null, null);
```

Comment:
actually the missing constructor is intentional, i don't think we need a constructor for everything as we have a setter...